src/js/components/Drop/doc.js

```diff
@@ -1,7 +1,17 @@
 import { describe, PropTypes } from 'react-desc';
 
+import { OVERFLOW_VALUES } from '../Box/doc';
 import { getAvailableAtBadge } from '../../utils';
 
+const dropOverflowPropTypes = PropTypes.oneOfType([
```

Comment:
Can we export this from Box/doc.js, since it seems to be identical?

Reply:
i tried. but when I use the object destruction it breaks the required prop. `{ ...OverflowPropTypes }`.

Reply:
Can we put a comment in then? That way, changing the Box prop types will be directed to align the Drop ones.